hashicorp/dev-portal
# chore(deps): update @hashicorp/react-product-downloads-page

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-BRANCH_NAME-hashicorp.vercel.app/PATH_TO_VIEW) 🔎
- [Asana task](url) 🎟️

## 🗒️ What

This PR bumps `@hashicorp/react-product-downloads-page` to remove bandwidth attributions.

## 🤷 Why



## 🛠️ How



## 📸 Design Screenshots



## 🧪 Testing

- For each product in the product switcher:
- Navigate to downloads page (from button in nav)
- Click through each OS platform and ensure "Bandwidth courtesy of" isn't displayed

## 💭 Anything else?
